tests/unit/model/state-test.js

```diff
@@ -138,7 +138,29 @@ for (let testRun = 0; testRun < 2; testRun++) {
         assert.equal(record.get('isLoaded'), true, 'record is loaded');
       });
       skip('isSaving', function () {});
-      skip('isDeleted', function () {});
+      test('isDeleted', function (assert) {
+        let data = {
+          data: {
+            id: 1,
+            type: 'com.example.bookstore.Book',
+            attributes: {
+              title: 'The Storm Before the Storm',
+              author: 'Mike Duncan',
+            },
+          },
+        };
+        let record = this.store.push(data);
+        assert.equal(record.get('isDeleted'), false, 'record starts off not deleted');
+        record.deleteRecord();
+        assert.equal(record.get('isDeleted'), true, 'record is now deleted');
+        record.rollbackAttributes();
+        assert.equal(
+          record.get('isDeleted'),
+          false,
+          'after rollbackAttributes record is no longer deleted'
+        );
```

Comment:
should we not write new tests with direct property access (i.e. what we expect our users to do) and not `get`?

Reply:
We still want to cover the older ember-data versions as well. I can open a PR that switches between `.get` and prop access for these tests based on version

Reply:
https://github.com/hjdivad/ember-m3/pull/1296

Reply:
@igorT makes sense; presumably something similar to https://github.com/hjdivad/ember-m3/blob/c384658c2cae1320c306e9152a955b5750e08fab/tests/helpers/schema-versions.js#L41-L78